source/renderer/app/components/staking/redeem-itn-rewards/Step1ConfigurationDialog.js

```diff
@@ -225,7 +225,7 @@ export default class Step1ConfigurationDialog extends Component<Props> {
   };
 
   get canSubmit() {
-    const { isSubmitting, wallet } = this.props;
+    const { isSubmitting, wallet, error, errorMessage } = this.props;
```

Comment:
@DeeJayElly can you unify these error messages? They have the same type, so the container could consolidate into a single one.

Reply:
Even if one of them requires `minRewardFunds` and the other doesn't, it's not an issue. You can always pass this prop.

Reply:
Fixed